api/openapi-spec/swagger.json

```diff
@@ -62256,6 +62256,10 @@
       "description": "Path within the container at which the volume should be mounted.  Must not contain ':'.",
       "type": "string"
      },
+     "mountPropagation": {
+      "description": "mountPropagation is the mode how are mounts in the volume propagated from the host to the container and from the container to the host. When not set, MountPropagationHostToContainer is used. This field is alpha in 1.8 and can be reworked or removed in a future release. Optional.",
```

Comment:
nit: "mountPropagation determines how mounts are propagated from the host to container and the other way around"

Reply:
fixed

Reply:
It looks like the text hasn't been updated yet?

Reply:
It is now, sorry.